NetworkConfigurations/RoutingAndForwarding/BGP_BorderGatewayProtocol.md

```diff
@@ -0,0 +1,674 @@
+| Route Advertisement                                                  | If a route is learned via iBGP, it is considered within its own AS. Routes Received from a peer cannot be advertised to another peer but can be advertised to an eBGP peer |                               If a route is learned via eBGP, it is understood as not belonging to the local AS. Routes received from a peer cna be broadcast to both eBGP nad iBGP peers                               |
+| Topology                                                             |                 Each iBGP speaker within AS must have all other iBGP peers in the same AS as neighbors. This configuration is called a Full Mesh Topology                  |                                                                                          A Full Mesh Topology is not required                                                                                           |
+| Loop Prevention                                                      |                  Uses BGP split horizon. This rule states that a BGP router does not advertise the routes learned from an iBGP peer to other iBGP peers.                   | The AS_Path attribute contains a list of numbers through which a route has traversed. If a router sees its AS number in the AS_PATH attribute of an incoming BGP packet, it will drop the advertisement to avoid loops. |
+| Default TTL between Peers                                            |                                                                                 TTL = 255                                                                                  |                                                                                                         TTL = 1                                                                                                         |
+| Default Administrative Distance (Used in determining route priority) |                                                                                    200                                                                                     |                                                                                                           20                                                                                                            |
+
+In this document, we will be using the open-source software
+FRR (Free Range Routing) to configure an example for BGP.
+
+For more information regarding configuring BGP outside of this document, visit the FRR Documentation:
+[BGP with FRR](https://docs.frrouting.org/en/latest/bgp.html#bgp)
+
+## Installation of FRR
```

Comment:
Have we confirmed that the installation and configuration of FRR is the same (or very similar) between BGP and OSFP documentation?

Reply:
Last time I checked, FRR was preinstalled in DENT images, so there shouldn't be any need to install it. Did this change?

Reply:
The VM version we are using does not have FRR installed. I will change the document to clarify this and explain the steps I took to install it on the VM.

Reply:
> Have we confirmed that the installation and configuration of FRR is the same (or very similar) between BGP and OSFP documentation?

I have made sure the BGP documentation has the installation steps used for FRR on the VMs detailed the same (or very similar) as the OSPF documentation.

Reply:
If those VM images are supposed to be DENT images, we might want to fix that.

If you are using an image with FRR preinstalled it might be confusing to see instructions to install FRR first, and may even cause weird issues if they decide to install a snap or docker image while the deb version is already installed.

In addition, it might make sense to have the instructions on how to install FRR on a separate page, to avoid duplicating it for every protocol.

Reply:
I will reach out to Sridhar to clarify the state of the images and make the suggested changes